packages/twenty-front/src/pages/settings/workspace/SettingsSubdomain.tsx

```diff
@@ -52,6 +56,11 @@ export const SettingsSubdomain = () => {
                 error={error?.message}
                 disabled={!!currentWorkspace?.customDomain}
                 fullWidth
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter') {
```

Comment:
Can we have an `onEnter` here or something less hardcoded? (maybe there are a few other instances of `(e.key === 'Enter')` in the codebase, we should also replace them. Thanks!

Reply:
I will do another PR to improve the management global of form.